src/react-disposable-decorator.js

```diff
@@ -1,10 +1,9 @@
 import React from "react";
 
-const inTestingEnv = typeof jasmine !== 'undefined';
-const enabled = !inTestingEnv || (typeof ReactDisposableDecoratorEnabled !== 'undefined' && ReactDisposableDecoratorEnabled);
+const disabled = typeof disableReactDisposableDecorator === 'undefined' || disableReactDisposableDecorator;
```

Comment:
`disableReactDisposableDecorator` might not be defined in the test environment, so we're defensive about checking its value to avoid throwing an error